### PR TITLE
server/cache: log updated region

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by huachaohuang** [comment]
PTAL @siddontang @disksing

**Comment by siddontang** [comment]
Do we need add log at L516 too, for the first insert.

**Comment by siddontang** [comment]
PTAL @disksing

**Comment by disksing** [comment]
Any need to log a message when leader is changed? cc @siddontang

**Comment by siddontang** [comment]
yes

**Comment by huachaohuang** [comment]
PTAL @disksing

File: server/cache.go [hunk 1]
```diff
@@ -531,6 +531,10 @@ func (c *clusterInfo) handleRegionHeartbeat(region *regionInfo) error {
 		return c.putRegionLocked(region)
 	}
 
+	if region.Leader.GetId() != origin.Leader.GetId() {
+		log.Infof("update region leader %v origin %v", region.Leader, origin.Leader)
```
Comment (disksing) [inline server/cache.go]: Need to log region id, or I think you may just log `region` and `origin` as above.

**Comment by disksing** [comment]
LGTM

**Pull request merged**